oasis-rpc/src/import/mod.rs

```diff
@@ -56,6 +48,42 @@ impl Importer {
     }
 }
 
+pub enum ImportLocation {
+    Path(std::path::PathBuf),
+    Url(url::Url),
+    // Address(oasis_types::Address),
```

Comment:
Is there a todo here?

Reply:
it's more of a note to readers that we have ambitions 😉